apis/pkg/v1alpha1/config.go

```diff
@@ -32,6 +32,12 @@ type ControllerConfigSpec struct {
 	// used to enable leader election process.
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty"`
+	// Docker image name.
+	// More info: https://kubernetes.io/docs/concepts/containers/images
+	// This field is optional to allow higher level config management to default or override
+	// container images in workload controllers like Deployments and StatefulSets.
+	// +optional
+	Image string `json:"image,omitempty"`
```

Comment:
nitpick: let's make this a `*string` so we evaluate if it is present by check on `nil` rather than length of string 👍